### PR TITLE
Getting started page for usability study

### PR DESCRIPTION
**Pull request by StevenBucher98**
# PR Summary

A getting started page that will be used for usability study and eventually the final doc pages.

### CODE REVIEW

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+### Azure Agent
+- "How do I create a new resource group with Azure CLI?"
+- "How can I list out the storage accounts I have in Azure PowerShell"
+- "What is Application Insights?"
+- "How to create a web app with Azure CLI?"
+
+Here is a quick demo showing the Azure Agent in action:
+
+![A Gif showing Azure Agent in action](/docs/media/AzureAgent.gif)
```
Comment (sdwheeler) [inline docs/GettingStarted.md:63]:
```suggestion
![An animation showing Azure Agent in action.](/docs/media/AzureAgent.gif)
```

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+### Inserting code
+
+When you are chatting with the agent, you can use the `/code post` command to automatically insert
+the code from the response into the working shell. This is a great way to quickly get the code you
+need to run in your shell. Additionally, you can use the hot key `Ctrl+d, Ctrl+d` to insert the code
+into the working shell.
+
+![A Gif showing Inserting Code with AIShell](/docs/media/InsertCode.gif)
```
Comment (sdwheeler) [inline docs/GettingStarted.md:98]:
```suggestion
When you are chatting with the agent, you can use the `/code post` command to automatically insert
the code from the response into the working shell. This is the simplest way to quickly get the code
you need to run in your shell. You can also use the hot key <kbd>Ctrl</kbd>+<kbd>d</kbd>,
<kbd>Ctrl</kbd>+<kbd>d</kbd> to insert the code into the working shell.

![An animation showing Inserting Code with AIShell.](/docs/media/InsertCode.gif)

```

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+AIShell supports key bindings for the `/code` command. They are currently hard-coded, but custom key
+bindings will be supported in future releases.
+
+| Key bindings              | Command          | Functionality |
+| ------------------------- | ---------------- | ------------- |
+| <kbd>Ctrl+d, Ctrl+c</kbd> | `/code copy`     | Copy _all_ the generated code snippets to clipboard |
+| <kbd>Ctrl+\<n\></kbd>     | `/code copy <n>` | Copy the _n-th_ generated code snippet to clipboard |
+| <kbd>Ctrl+d, Ctrl+d</kbd> | `/code post`     | Post _all_ the generated code snippets to the connected application |
+| <kbd>Ctrl+d, \<n\></kbd>  | `/code post <n>` | Post the _n-th_ generated code snippet to the connected application |
```
Comment (sdwheeler) [inline docs/GettingStarted.md:110]:
```suggestion
AIShell has key bindings for the `/code` command. They key bindings are currently hard-coded, but
custom key bindings will be supported in future releases.

|                       Key bindings                       |     Command      |                            Functionality                            |
| -------------------------------------------------------- | ---------------- | ------------------------------------------------------------------- |
| <kbd>Ctrl+d</kbd><kbd>Ctrl</kbd>+<kbd>c</kbd>            | `/code copy`     | Copy _all_ the generated code snippets to clipboard                 |
| <kbd>Ctrl</kbd>+<kbd>\<n\></kbd>                         | `/code copy <n>` | Copy the _n-th_ generated code snippet to clipboard                 |
| <kbd>Ctrl</kbd>+<kbd>d</kbd><kbd>Ctrl</kbd>+<kbd>d</kbd> | `/code post`     | Post _all_ the generated code snippets to the connected application |
| <kbd>Ctrl</kbd>+<kbd>d</kbd><kbd>\<n\></kbd>             | `/code post <n>` | Post the _n-th_ generated code snippet to the connected application |
```

File: docs/GettingStarted.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+### Resolving Errors
+
+If you encounter an error in your working terminal, you can use the `Resolve-Error` cmdlet to send
+that error to the open AIShell window for resolution. This will allow you to get a response from the
+AI model open to help you resolve the error.
```
Comment (sdwheeler) [inline docs/GettingStarted.md:116]:
```suggestion
If you encounter an error in your working terminal, you can use the `Resolve-Error` cmdlet to send
that error to the open AIShell window for resolution. This command asks the AI model to help you
resolve the error.
```

File: docs/GettingStarted.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+- How do I get the top 10 most CPU intensive processes on my computer?
+
+Azure Agent
+
+- "How do I create a new resource group with Azure CLI?"
+- "How can I list out the storage accounts I have in Azure PowerShell"
+- "What is Application Insights?"
+- "How to create a web app with Azure CLI?"
+
+Here is a quick demo showing the Azure Agent in action:
```
Comment (mikefrobbins) [inline docs/GettingStarted.md:65]:
```suggestion
Here's a quick demo showing the Azure Agent in action:
```

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
See my comments

**Review by daxian-dbw — APPROVED** [review]

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**